README.md

```diff
@@ -13,42 +13,46 @@ JWST Calibration Pipeline
 Installation
 ------------
 
-The ``jwst`` package can be installed into a virtualenv or conda environment via ``pip``.  We recommend creating a fresh environment with only python installed.  If using conda, make sure you have a recent version of Anaconda or Miniconda installed.  Via conda:
+The `jwst` package can be installed into a virtualenv or conda environment via `pip`.  We recommend creating a fresh environment with only python installed.  If using conda, make sure you have a recent version of Anaconda or Miniconda installed.  Via conda:
 
-    conda create -n jwst_env python=3.7
+    conda create -n jwst_env python
```

Comment:
We don't need to pin the python version any more?

Reply:
Right.  The package and its dependencies all support python 3.8.  That said, the DMS distribution still installs python 3.7.6, as all the packages are pinned.